src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -238,7 +250,16 @@ public bool PreviousSectionCompleted(int sequenceId, int sectionId)
                 {
                     return true;
                 }
-                return (FinishSectionStatus(sectionId-1) == TaskListSectionStatus.Completed);
+
+                // TODO: APR-1193 - I have added the check for the case of section 9.3 being NotRequired for Supporting Provider
```

Comment:
This is a brittle solution that would need code changes in Apply if the sections were re-organised in QnA. It would be simpler to check for previous sections having a status of Completed or Not Required

Reply:
Checking the status Completed or Not Required in this case is not enough. We have to take into consideration Supporting Provider check. And the same approach is used for all other case. 
Any refactoring should be done for the whole methods, which will involves significant amount of coding and regressive testing.

Reply:
See https://skillsfundingagency.atlassian.net/browse/APR-1356 
This needs to be properley resolved ASAP.
For now I will approve fior testing.